pymtl3/stdlib/test/test_sinks.py

```diff
@@ -9,7 +9,7 @@
 """
 
 from pymtl3 import *
-from pymtl3.stdlib.ifcs import RecvIfcRTL, RecvRTL2SendCL, enrdy_to_str
+from pymtl3.stdlib.ifcs import RecvIfcRTL, RecvRTL2SendCL
```

Comment:
We talked about the plan of moving `stdlib.test` to `stdlib.test_utils` or whatever we feel is appropriate because `test` sounds like it is the directory for test cases?

Reply:
i forgot about that. Thanks for the reminder. Let me fix them.